run/markdown-preview/e2e_test_setup.yaml

```diff
@@ -0,0 +1,57 @@
+steps:
```

Comment:
nit: format this file similar to https://github.com/GoogleCloudPlatform/cloud-code-samples/blob/master/python/.ci/cloud-run-python-hello-world.cloudbuild.yaml

Reply:
Took a stab at it. LMK if you want any changes.